src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -330,10 +334,24 @@ private static bool GetForwardAuditMessages()
             throw new Exception("ForwardAuditMessages settings is missing, please make sure it is included.");
         }
 
+
+        public static Type GetType(string typeName)
```

Comment:
is this necessary? Couldn't we just brute force try to load the type that is passed to us? Which would then automatically load it into the appdomain?

Reply:
I added this because the brute force didn't work :(